src/IterableCodeExtractor.php

```diff
@@ -65,6 +66,17 @@ public static function fromFile( $file_or_files, Translations $translations, arr
 				}
 			}
 
+			if ( ! empty( $options['wpExtractPatterns'] ) ) {
+				$headers = MakePotCommand::get_file_data_from_string( $string, [ 'Title' => 'Title' ] );
```

Comment:
Need to check what other metadata we need to translate. According to https://github.com/WordPress/gutenberg/pull/36751/files#diff-ef23af8c1b0d6931bd6bbedd91dcfa92b5537a438c1c0c45499b8efdf2c91546R84 we may need to consider `Description` as well.

Reply:
_Description_ definitely sounds like something user-facing that needs i18n :-)

Reply:
Description added at https://github.com/wp-cli/i18n-command/pull/312/commits/cc4b02b4e3de8dde18e9f5f91d3c1e716c99fab7